cve_bin_tool/parsers/python.py

```diff
@@ -28,16 +28,27 @@ def run_checker(self, filename):
                     "--report",
                     "-",
                     "--quiet",
-                ]
+                ],
+            )
+        except subprocess.CalledProcessError:
+            self.logger.error(
+                f"{filename} not scanned: pip --dry-run was unable to get package versions."
+            )
+            self.logger.error(
+                "pip version >= 22.2 is required to scan python requirements files"
             )
-        )
-        for line in lines["install"]:
-            product = line["metadata"]["name"]
-            version = line["metadata"]["version"]
-            vendor = self.find_vendor(product, version)
-            if vendor is not None:
-                yield from vendor
-        self.logger.debug(f"Done scanning file: {self.filename}")
+
+        else:
+
+            lines = json.loads(output)
+
```

Comment:
This is some generous spacing, I think it's perfectly readable with less.
```suggestion
        else:
            lines = json.loads(output)
```